profiler/README.MD

```diff
@@ -34,6 +34,8 @@ The reproducer will also write a `repro.py` script that runs a single computatio
 
 Example command line: `python external_profiler.py --stencil=MY_STENCIL <TARGET PYTHON SCRIPT>.py <TARGET ARGS>`
 
+If you move the `repro_XXXX` folder you'll have to adapt the hardcoded path from the py wrapper to the `.so` that are embedded in the `original_code` folder. Search for `pyext_module = gt_utils.make_module_from_file` in the `original_code`.
```

Comment:
I feel like it would be nice to say this more explicitly, except most of the folder structure changes from run to run. Maybe something like:
```suggestion
The `.py` file in the generated `original_code` directory includes `pyext_module = gt_utils.make_module_from_file`, which uses a hardcoded path to the generated `.so` files. If you move the `repro_XXXX` folder you'll have to change this hardcoded path.
```